src/Rector/Contrib/PHPUnit/DelegateExceptionArgumentsRector.php

```diff
@@ -29,6 +29,16 @@
         'setExpectedExceptionRegExp' => 'expectExceptionMessageRegExp',
     ];
 
+    /**
+     * @var string[]
+     */
+    private $phpUnitTestCaseClasses = [
+        // PHPUnit 6 and lower
```

Comment:
"PHPUnit 4 and lower"

Reply:
I think it is overlapping BC support even in PHPUnit 6.0
Could you check when it was really removed?

Reply:
No, [PHPUnit 6.0 is complete namespaced](https://github.com/sebastianbergmann/phpunit/blob/6.0/ChangeLog-6.0.md#changed-1) now :smile:

Reply:
I will update those to be less confusing